modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/init.ftl

```diff
@@ -230,51 +229,13 @@
 	/>
 </#if>
 
-<#assign the_title = "" />
-
```

Comment:
I'm happy with removing this bit as it's a complex logic. However, if we remove `the_title` variable then we need to provide another value for it in some other `unstyled` theme ftl snippets in [portal_normal.ftl](https://github.com/liferay/liferay-portal/blob/03e5fca7aefe15eb0133034dfe6d729778773c4c/modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/portal_normal.ftl#L55) and [portal_pop_up.ftl](https://github.com/liferay/liferay-portal/blob/ab7b964caeba85b0381dcca61b190806af18e45f/modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/portal_pop_up.ftl#L8). 

These end up in classic and admin themes

Even if we fix this for our own themes, I'm a bit hesitant to remove the variable as it may be used in custom themes. We should either document this as a breaking change or provide a fallback where `the_title` is declared and set to be `html_title`

Reply:
Ok, no problem, I'm going to remove this commit.